### PR TITLE
License and supports in django admin

### PR DESCRIPTION
**Pull request by szok**
Added:
- License admin fieldsets
- Support admin fieldsets
- Fix Calendar icon
- AssetModel and service env to django admin

Changed:
- New mechanism to generate permissions

Connected to #1445

### CODE REVIEW

File: src/ralph/licences/models.py [hunk 1]
```diff
@@ -36,8 +37,11 @@ class LicenceType(NamedMixin, models.Model):
     def create_from_string(cls, string_name, *args, **kwargs):
         return cls(name=string_name)
 
+    class Permissions:
+        blacklist = set()
```
Comment (mkurek) [inline src/ralph/licences/models.py]: do we really need to add this to every class which inherits from PermByFieldMixin?

Comment (szok) [inline src/ralph/licences/models.py]: Ok, I will add exception as there is no defined class.

Comment (ar4s) [inline src/ralph/licences/models.py]: @mkurek why not?

Comment (mkurek) [inline src/ralph/licences/models.py]: do you add empty `Meta` or 

```
class Meta:
    unique_together = []
```

to every model? Empty `blacklist` could be default option - only when you want to blacklist sth you could add `Permissions` class to model definition.

Comment (ar4s) [inline src/ralph/licences/models.py]: Now I see :smile: I thought it concern `lib/permissions/models.py` 
Maybe that way https://github.com/django/django/blob/master/django/db/models/base.py#L83-L84

File: src/ralph/lib/permissions/models.py [hunk 2]
```diff
@@ -56,7 +56,10 @@ def __new__(cls, name, bases, attrs):
         )
         class_name = new_class._meta.model_name
         for field in new_class._meta.fields:
-            blacklist = new_class.Permissions.blacklist
+            try:
```
Comment (mkurek) [inline src/ralph/lib/permissions/models.py:70]: you could move this before for loop to check for blacklist only once

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**